models/sessions/scratch/bigquery/snowplow_web_sessions_this_run.sql

```diff
@@ -193,7 +174,7 @@ session_aggs as (
                 ))  +
             -- number of page pings following a page view (or no event) (gap of min visit length)
             (count(distinct case when event_name = 'page_ping' and page_view_id is not null then page_view_id else null end) * {{ var("snowplow__min_visit_length", 5) }}) as engaged_time_in_s,
-        {{ snowplow_utils.timestamp_diff('min(derived_tstamp)', 'max(derived_tstamp)', 'second') }} as absolute_time_in_s
+        {{ snowplow_utils.timestamp_diff('min(derived_tstamp)', 'max(derived_tstamp)', 'second') }} as absolute_time_in_s,
```

Comment:
The conversions macro prefixes commas so this shouldn't be necessary - really we need to swap the whole model to leading comma rather than trailing, but we should do that in a standalone PR once it's all stable... I'll create an issue to do this so we don't forget